lib/dictator/plug.ex

```diff
@@ -24,9 +24,15 @@ defmodule Dictator.Plug.Authorize do
     resource_key = opts[:resource_key] || :current_user
     user = conn.assigns[resource_key]
     action = conn.private.phoenix_action
-    resource = apply(policy, :load_resource, [conn.params])
 
-    if apply(policy, :can?, [user, action, resource]) do
+    target =
+      if policy.resourceful?() do
```

Comment:
what you want is to declare module attributes, not a function that is just meant to let you know what type of policy this is

I just haven't touched this part yet. Would prefer to refactor that later. My goal here was to first get this to a state that could work for my project

Reply:
Hm, that was my first alternative but it feels hidden to developers. However, I think it would be only internal to the lib since 99.9% of the use cases would be covered by expanding the `Bare` and `Simple` policies. What are your thoughts?

Reply:
Also, for this change I'd rather have it in a single PR that implements the `Bare` policy (or equivalent name), so if possible can you do it here?

Reply:
this seems to be 100% for internal use at this point

also, a common pattern here is to declare a behaviour (even if empty), and people can get module metadata to see if it implements the resourceful behaviour